styleguide/derek/_partials/solutions/announcements.ejs

```diff
@@ -0,0 +1,53 @@
+<div class="container-fluid standard-padding bg-light-gray">
+  <div class="container">
+
+    <div class="center-block">
+      <h2 class="center teal">Announcements</h2>
+    </div>
+
+    <div class="announcement-container announcement-offer row-eq-height">
+      <a href="#" class="announcement-close"></a>
+      <div class="announcement-icon"></div>
+      <div class="announcement-message">
+        <p>Lorem ipsum dolor sit amet, cum cras lectus consectetuer dictum.
+        Suscipit porta, enim id metus dicta ultrices, id orci urna, adipiscing
+        lectus mattis. Tristique nibh et, consectetuer faucibus cras nec mollis
+        vehicula etiam, sit eu luctus facilisi tincidunt a.</p>
+      </div>
+    </div>
+
+    <div class="announcement-container announcement-warning row-eq-height">
+      <a href="#" class="announcement-close"></a>
+      <div class="announcement-icon"></div>
+      <div class="announcement-message">
+        <p>Lorem ipsum dolor sit amet, cum cras lectus consectetuer dictum.
+        Suscipit porta, enim id metus dicta ultrices, id orci urna, adipiscing
+        lectus mattis. Tristique nibh et, consectetuer faucibus cras nec mollis
+        vehicula etiam, sit eu luctus facilisi tincidunt a.</p>
+      </div>
+    </div>
+
+    <div class="announcement-container announcement-danger row-eq-height">
+      <a href="#" class="announcement-close"></a>
+      <div class="announcement-icon"></div>
+      <div class="announcement-message">
+        <p>Lorem ipsum dolor sit amet, cum cras lectus consectetuer dictum.
+        Suscipit porta, enim id metus dicta ultrices, id orci urna, adipiscing
+        lectus mattis. Tristique nibh et, consectetuer faucibus cras nec mollis
+        vehicula etiam, sit eu luctus facilisi tincidunt a.</p>
+      </div>
+    </div>
+
+    <div class="announcement-container announcement-success row-eq-height">
+      <a href="#" class="announcement-close"></a>
+      <div class="announcement-icon"></div>
+      <div class="announcement-message">
+        <p>Lorem ipsum dolor sit amet, cum cras lectus consectetuer dictum.
+        Suscipit porta, enim id metus dicta ultrices, id orci urna, adipiscing
+        lectus mattis. Tristique nibh et, consectetuer faucibus cras nec mollis
+        vehicula etiam, sit eu luctus facilisi tincidunt a.</p>
```

Comment:
Do you mind including one of these with very little content and one with a lot of content please?